cmd/spicedb/main.go

```diff
@@ -28,9 +26,6 @@ const (
 var errParsing = errors.New("parsing error")
 
 func main() {
-	// Set up a seed for randomness
-	rand.Seed(time.Now().UnixNano())
```

Comment:
why is this being removed?

Reply:
`rand.Seed` is deprecated as of 1.20, so our linters will not be happy. I decided to address the problem. Essentially we don't need to do this anymore if we use `crypto/rand` instead of `math/rand`